generators/generatorHelper.js

```diff
@@ -0,0 +1,67 @@
+const timeBoundSuffixes = ['', '_1d', '_7d', '_30d', '_60d', '_90d', '_180d', '_365d', '_2y', '_3y', '_5y', '_10y']
+
+const data = [
+  { metric: 'daily_avg_marketcap',
+    description: 'Gets the daily average marketcap',
+    fiatCurrency: '_usd',
+    timeBound: ['']
```

Comment:
Don't add empty timeBound or fiatCurrency, check if the key exists in the function generation.

Reply:
Okay, I did with if statements, it should be okay now